### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by nanaten**
Update README.md.
Add LiveData and View implementations.
Add merge, filter, map, combineLatest documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,4 +75,51 @@ int.toStringWithSeparator(locale = Locale.JAPAN)
...
+liveData.value = true // emitting once, and don't hold a value.
+```
+
+`LiveData<T>.filter` filter LiveData that emits value by predicate.
+
+```kotlin
+liveData.filter { it == true }
+```
+
+`merge` merge any number of `LiveData` that emit a value, Emit latest values.
```
Comment (okuzawats) [inline README.md:103]: I think it should be like this...
```suggestion
`merge` merges any number of `LiveData` and emits latest values.
```

File: README.md [hunk 2]
```diff
@@ -71,4 +75,51 @@ int.toStringWithSeparator(locale = Locale.JAPAN)
...
+
+```kotlin
+val mergedData = merge(liveData1, liveData2)
+liveData1.value = true // mergedData is emitted `true`.
```
Comment (okuzawats) [inline README.md:107]: maybe this makes sense more🤔
```suggestion
liveData1.value = true // mergedData emits `true`.
```

File: README.md [hunk 3]
```diff
@@ -71,4 +75,51 @@ int.toStringWithSeparator(locale = Locale.JAPAN)
...
+```
+
+`map` converts `LiveData` the type parameter `T` to `R`.
+
+```kotlin
+liveData.value = 123
+val convertedData = liveData.map { it.toString() } // convertedData is the "123" of String value.
```
Comment (okuzawats) [inline README.md:114]: same here...
```suggestion
val convertedData = liveData.map { it.toString() } // convertedData emits "123", String type.
```

File: README.md (hunk 1 above)
Comment (nanaten) [reply README.md]: 👍

File: README.md (hunk 2 above)
Comment (nanaten) [reply README.md:107]: 👌

File: README.md (hunk 3 above)
Comment (nanaten) [reply README.md:114]: 🙇

File: README.md [hunk 4]
```diff
@@ -71,4 +75,51 @@ int.toStringWithSeparator(locale = Locale.JAPAN)
...
+```
+
+`combineLatest` combine some `LiveData` and returns a new value.
+
+```kotlin
+val combinedData = combineLatest(liveData1, liveData2) { live1, live2 ->
+  "${live1} and ${live2}"
+}
+liveData1.value = "test"
+liveData2.value = true // combinedData is the "test and true".
```
[the remark this answers is not in the document]
Comment (nanaten) [reply README.md:124]: 👍

**Review by okuzawats — APPROVED** [review]
🎉

**Comment by nanaten** [comment]
🚀

**Pull request merged**